confgenerator/logging_processors.go

```diff
@@ -132,10 +133,10 @@ func (p LoggingProcessorParseRegex) Components(tag, uid string) []fluentbit.Comp
 	parser.Config["Format"] = "regex"
 	parser.Config["Regex"] = p.Regex
 
-	return []fluentbit.Component{
-		parser,
-		fluentbit.ParserFilterComponent(tag, p.Field, []string{parserName}),
-	}
+	parserFilters := []fluentbit.Component{}
```

Comment:
Any reason this can't be done as one literal slice creation?
i.e.
```
return []fluentbit.Component{
    parser,
    fluentbit.ParserFilterComponents(tag, p.Field, []string{parserName}, false)...
}
```